README.md

```diff
@@ -17,182 +17,42 @@ This plugin can be installed using `luarocks`.
-
-The required `match` field is a regular expression, implicitly anchored to the beginning of the URI value.
-
-The optional `replacement` field is used to provide an updated value for the resource name.
-
-`--data 'config.resource_name_rule[1].match=/api/v1/users'`
-
-`--data 'config.resource_name_rule[2].match=/api/v1/features/xyz/enabled' --data 'config.resource_name_rule[2].replacement=/api/v1/features/?/enabled'`
-
-The first matching rule in a list of rules is used, and any remaining rules are ignored.
-So if a rule matching `/api` exists before a more-specific match like `/api/v1/users`, the `/api` rule will be used.
-
-Example setup:
-```
---data 'config.resource_name_rule[1].match=/api/v1/users'
---data 'config.resource_name_rule[2].match=/api/v1/features/\w*/enabled' --data 'config.resource_name_rule[2].replacement=/api/v1/features/?/enabled'
---data 'config.resource_name_rule[3].match=/reset_password/' --data 'config.resource_name_rule[3].replacement=PASSWORD_RESET'
---data 'config.resource_name_rule[4].match=/([^/]*)/' --data 'config.resource_name_rule[4].replacement=/$1/?
+# Enabled for specific service only
+curl -i -X POST --url http://${KONG_ADMIN_HOST}:${KONG_ADMIN_PORT}/services/example-service/plugins/ --data 'name=ddtrace'
 ```
 
-Example outcomes:
-| HTTP Method | Request URI | Matches Rule | Final Resource Name |
-| ----------- | ----------- | ------------ | ------------------- |
-| GET | /api/v1/users/1234/profile | 1 | GET /api/v1/users |
-| GET | /api/v1/features/abc/enabled | 2 | GET /api/v1/features/?/enabled |
-| GET | /api/v1/features/xyz/enabled | 2 | GET /api/v1/features/?/enabled |
-| POST | /reset_password/D6T6wVRw | 3 | POST PASSWORD_RESET |
-| GET | /static/site.js | 4 | GET /static/? |
-| GET | /favicon.ico | none | GET /favicon.ico |
-
-
-### Tag root span with HTTP Headers (DD_TRACE_HEADER_TAGS)
-
-For security reasons, only a subset of HTTP headers are reported as tags. It is possible to configure the plugin to report specific HTTP Headers as span tags.
-Nonetheless, be careful on which headers you are deciding to add as a span tag.
-
-Learn more about [HTTP header collection](https://docs.datadoghq.com/tracing/configure_data_security/?tab=net#collect-headers)
-
-Example setup:
-
-````sh
-# curl
---data 'config.header_tags[1].header=Ratelimit-Limit' --data 'config.header_tags[1].tag=rate-limit'
-
-# The tag can be omitted, a tag following this format will be used: `http.<request|response>.headers.<http-header-name>`
---data 'config.header_tags[1].header=Ratelimit-Limit'
+Kong DB-less:
+````yaml
+# Enable for a specific service
+_format_version: "3.0"
+_transform: true
+
+services:
+- name: example-service
+  url: http://httpbin.org/headers
+  plugins:
+  - name: ddtrace
+    config:
+      service_name: example-service
+      agent_host: datadog-agent
```

Comment:
Would be possible to show the full configuration of this plugin in this example? Like the use of `static_tags` for example? 
Showing all the possibilities makes it easier to not break the configuration with a new input.

Reply:
@dmehala works for me.